src/js/core.js

```diff
@@ -339,17 +339,14 @@
             element.setAttribute('role', 'textbox');
             element.setAttribute('aria-multiline', true);
             element.setAttribute('medium-editor-index', index);
+        }, this);
 
-            if (element.hasAttribute('medium-editor-textarea-id')) {
```

Comment:
I think you shouldn't remove this condition, it will avoid an extra listener when user doesn't use MediumEditor with textarea.